src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js

```diff
@@ -215,7 +215,15 @@
             return [...relationsContainer.querySelectorAll('[type="checkbox"]')];
         };
         const attachRowsEventHandlers = () => {
-            findOrderInputs().forEach((item) => item.addEventListener('blur', updateSelectedItemsOrder, false));
+            findOrderInputs().forEach((item) => {
+                item.addEventListener('blur', updateSelectedItemsOrder, false);
+                item.addEventListener('change', focusOnElement, false);
+            });
+        };
+        const focusOnElement = (event) => {
+            if(doc.activeElement !== event.target) {
```

Comment:
please install Prettier and use it on the file

Reply:
Done